torch/ao/quantization/backend_config/native.py

```diff
@@ -700,6 +699,43 @@ def get_native_backend_config_dict():
         default_op_quint8_dtype_config,
         default_op_fp16_dtype_config
     ]
+    fixed_qparams_op_dtype_configs = [
+        weighted_op_int8_dtype_config,
+        default_op_fp16_dtype_config,
+    ]
+    return {
+        # optional
+        "name": "_native_and_fp16",
+        "configs": [
+            *_DEFAULT_OP_INT8_CONFIGS,
+            *_get_linear_configs(linear_dtype_configs),
+            *_get_conv_configs(conv_dtype_configs),
+            *_get_binary_op_configs(binary_op_dtype_configs),
+            *_get_fixed_qparams_op_configs(fixed_qparams_op_dtype_configs),
+            _CAT_CONFIG,
+            *_get_bn_configs(),
+            *_get_share_qparams_op_configs(share_qparams_op_dtype_configs),
+            *_get_rnn_op_configs(),
+            *_get_embedding_op_configs(),
```

Comment:
I think in this case it may be worth it in case we want to add/remove ops to this list, e.g. if we forget to modify one of these two places then the resulting behavior may be difficult to debug. Not a strong preference though. Either is fine with me.

Reply:
seems like a tradeoff of optimizing for reading existing code versus optimizing for writing new code.  I'd estimate the "easy to read" use case to be more important for this instance.

In fact, I'd argue that removing a lot of the utilities in this file and replacing it with dead simple and duplicated configs might be valuable in the future.  Currently, this file seems optimized for writes but it really difficult to read.

Reply:
I'd love to hear ideas on how to both enforce consistency for adding new things as well as keep things readable.  If we can only pick one, I think picking readability is a better choice in this case.